sdk/keyvault/keyvault-admin/tests.yml

```diff
@@ -10,6 +10,7 @@ extends:
       # instances per region per subscription) so we're only running
       # live tests against a single instance.
       Location: eastus2
+      PersistOidcToken: true
```

Comment:
Do we need this for the Keys weekly tests as well?

Reply:
We sure do! Thanks Timo 👍